saferpay_test/src/Controller/SaferpayTestController.php

```diff
@@ -29,29 +32,46 @@ public function __construct() {
   /***
    * With CreatePayInit() a payment link can be generated.
    *
-   * @return RedirectResponse
+   * @param Request
+   *
+   * @return Response
    */
-  public function createPayInit() {
-    $data = $_GET;
-
-    return new RedirectResponse(Url::fromRoute('saferpay_test.saferpay_test_form'));
+  public function createPayInit(Request $request = NULL) {
+    return new Response(Url::fromRoute('saferpay_test.saferpay_test_form', array(), array(
+      'query' => $request->query->all()))->setAbsolute()->toString());
   }
 
+
+
   /**
    * Verifies the digital signature of the confirmation message (MSGTYPE=PayConfirm)
```

Comment:
The first line in a docblock should be a single line. Possibly move this to a description below that, and make the first line just "Simulates the Saferpay PayConfirm endpoint.".